R-packages/modeltools/R/matrix.R

```diff
@@ -0,0 +1,72 @@
+#' Create a training and prediction matrices for a given ahead.
+#'
+#' @param lagged_df Data frame of lagged data.  It should have the following columns:
+#'     \itemize{
+#'     \item{`geo_value`}{Strings of geographic locations.}
+#'     \item{`time_value`}{Dates of training data.}
+#'     \item{Covariate columns}{Columns with names of the form `value-{days}:{signal}` or
+#'         `value+0:{signal} whose values correspond to `{signal}` `{days}` before `time_value`}
+#'     \item{Response columns}{Columns with names of the form `response+{n}:{response}` whose values
+#'         correspond to `{response}` `{n}` incidence period units after `time_value`.}
+#'     }
+#'     A data frame in this format can be made using `covidcast::aggregate_signals()`.
```

Comment:
I think it is no longer true that `covidcast::aggregate_signals`` gives this because of the new naming convention.

Reply:
Updated to include the new `add_response_columns()` function.